rs/ic_os/build_tools/partition_tools/src/ext.rs

```diff
@@ -190,6 +194,31 @@ impl Partition for ExtPartition {
 
         Ok(())
     }
+
+    async fn copy_file_to(&mut self, input: &Path, output: &Path) -> Result<()> {
```

Comment:
@frankdavid What do you think of this trait method? The impl for `fat` makes a little more sense since we can `mcopy` directly... but this is mostly just reading/writing from memory, which could easily be handled by the user instead.